### PR TITLE
updating README

### PR DESCRIPTION
**Pull request by cdsap**
Readme Update with the information about the new plugins. 

Part of #243

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +32,10 @@ _"... while some certainly had a defensive purpose, the use of others is not cle
 https://en.wikipedia.org/wiki/Talaiot
 
 # Table of Contents
-1. [Setup Plugin](#setup-plugin)
+1. [Setup Plugin](#setup)
```
Comment (mokkun) [inline README.md:35]: Should we use only "Setup" like [in the title of this item](https://github.com/cdsap/Talaiot/pull/253/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R47)?

File: README.md [hunk 2]
```diff
@@ -171,6 +315,7 @@ Talaiot will send to the InfluxDb server defined in the configuration the values
 | publishBuildMetrics          | Publish build metrics of the publisher, true by default                             |
 | publishTaskMetrics           | Publish tasks metrics of the publisher, true by default                             |
 
+Included in: `com.cdsap.talaiot` and `com.cdsap.talaiot.plugin.influxdb` plugins
```
Comment (mokkun) [inline README.md:318]: Nit: all "Included in" sections in the document miss a period.
```suggestion
Included in: `com.cdsap.talaiot` and `com.cdsap.talaiot.plugin.influxdb` plugins.
```

File: README.md [hunk 3]
```diff
@@ -494,17 +558,17 @@ Once is finished you can check the results on the Grafana Dashboard http://local
...
 
-![](resources/taskcache.png) 
+![](resources/taskcache.png)
 
 http://localhost:3003/d/WlpZEBRMz/task-cache-info?orgId=1
 
 
 
-## Other Plugins  <a name="plugins"></a>
+## Other Existing libraries  <a name="plugins"></a>
```
Comment (mokkun) [inline README.md:567]: Nit: for consistency in this title, we should either do:
```suggestion
## Other Existing Libraries  <a name="plugins"></a>
```
Or:
```suggestion
## Other existing libraries  <a name="plugins"></a>
```

File: README.md [hunk 4]
```diff
@@ -23,10 +32,10 @@ _"... while some certainly had a defensive purpose, the use of others is not cle
...
 
 # Table of Contents
-1. [Setup Plugin](#setup-plugin)
+1. [Setup Plugin](#setup)
 2. [Snapshots](#snapshots)
 3. [Basic Configuration](#basic-configuration)
-4. [Talaiot Extension](#talaiot-extension)
+4. [Talaiot](#talaiot-extension)
 5. [Example: Analyzing Data provided by Talaiot](#example)
 6. [Other Plugins](#plugins)
```
Comment (mokkun) [inline README.md:40]: [Title has changed](https://github.com/cdsap/Talaiot/pull/253/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R567) to "Other Existing Libraries". Should we update it in here as well?

File: README.md (hunk 3 above)
Comment (tinder-inakivillar) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (tinder-inakivillar) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (tinder-inakivillar) [reply README.md]: done

File: README.md [hunk 5]
```diff
@@ -23,10 +32,10 @@ _"... while some certainly had a defensive purpose, the use of others is not cle
 https://en.wikipedia.org/wiki/Talaiot
 
 # Table of Contents
-1. [Setup Plugin](#setup-plugin)
+1. [Setup Plugin](#setup)
 2. [Snapshots](#snapshots)
 3. [Basic Configuration](#basic-configuration)
-4. [Talaiot Extension](#talaiot-extension)
+4. [Talaiot](#talaiot-extension)
```
[the remark this answers is not in the document]
Comment (tinder-inakivillar) [reply README.md]: You're right, thanks,

File: README.md (hunk 2 above)
Comment (cdsap) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (cdsap) [reply README.md]: done

**Comment by cdsap** [comment]
thanks for the review, I'm thinking in release on Saturday, what do you think? We can always merge the sample later.

**Comment by mokkun** [comment]
> thanks for the review, I'm thinking in release on Saturday, what do you think? We can always merge the sample later.

Sounds good to me. :+1:

**Pull request merged**